input/pagecontent/profiles-and-extensions.md

```diff
@@ -11,13 +11,18 @@ The following is the full list of profiles that have been defined for this imple
 
 ### Extensions
 
-The following [extensions]({{site.data.fhir.path}}extensibility.html) have been defined as part of this guide. A [registry of standard extensions]({{site.data.fhir.path}}extensibility-registry.html) can be found in the FHIR specification and additional extensions may be registered on the HL7 FHIR registry at <http://hl7.org/fhir/registry>.
+All [extensions]({{site.data.fhir.path}}extensibility.html) used in this guide are defined in the FHIR Extensions Pack or [AU Base](https://build.fhir.org/ig/hl7au/au-fhir-base/profiles-and-extensions.html#extensions).
 
+The following extensions are marked with Must Support and referenced by this implementation guide:
+
+* [Australian Indigenous Status](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-indigenous-status.html) in [AU Core Patient](http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient)
+* [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) in [AU Core Patient](http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient)
+* [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) in [AU Core Patient](http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient)
 
 <!-- ================================================ -->
 <!--  use this line to include an autogenerated list of all profiles and highlight new ones using the input/data/new_stuff.yml list.  Remove it if you would like to hand generate it -->
 
-{% include ext-list-generator.md %}
+<!-- {% include ext-list-generator.md %}
```

Comment:
Is there missing close tag on this comment?

Reply:
Yes, there is! It still renders fine but we should fix it.

Reply:
Including close tag caused it to render in IG - possibly due to issue in template. 
Content deleted as currently not required, and can be added back as needed.